### DISCUSSION TITLE
Proposed pull requests and some questions

### DISCUSSION THREAD

**Discussion by Phelra** [aleksilassila/reiverr#114] [General]
Congratulations on all the work; this tool is shaping up to be very promising! I've taken the liberty of submitting a pull request to enhance language support on the web interface. I wanted to contribute to the project, though I admit my skills might not be top-notch. I hope find it useful. I noticed you're currently focused on the application for smart TVs (Android TV). I was considering posting some "bugs" or suggestions for improvement, but given your current focus on the TV app, perhaps now might not be the best time?

**Comment by aleksilassila** [comment]
Thank you! I really appreciate the thought, but right now it's indeed not the best time as the tv app rework is planned to replace the old web interface as well. This means that once I get it into a usable state, pretty much the whole codebase will be replaced, along with any bugs.

I'll try to get your pr merged soon though, and if you like to contribute before the rework despite the fact, you can. The old interface will always be available for download, but it probably won't get further updates from me. 

Cheers!

**Reply by Phelra** [reply]
I understand very good idea, myself I use mainly on an android tv :) In that case, I'll wait for the new version to make new
contributions. See you soon!

**Reply by GrabbenD** [reply]
> myself I use mainly on an android tv

How do you use Reiverr on Android TV?
WebView + Air Mouse?

**Reply by Phelra** [reply]
I wanted to say that I currently use the jellyfin application directly on my android tv. I don't know the equivalent of jellyseerr on this type of platform. 😅
